### ISSUE TITLE
Error when using suggested tarball URL path

### ISSUE THREAD

**Issue by kahmali** [DispatchMe/mgp#13]
In the docs it suggests adding a git repo like this (in `git-packages.json`):
 
```json
{
  "my:private-package": {
    "tarball": "https://api.github.com/repos/my/private-packages/tarball/commithash",
    "path": "optional/directory/path"
  },
  "my:other-private-package": {
    "tarball": "https://api.github.com/repos/my/private-packages/tarball/commithash"
  },
  "token": "GITHUB_ACCESS_TOKEN"
}
```

However, in some repos, I get the following error whenever I try to use the suggested `tarball` URL path:

```bash
events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: incorrect header check
    at Zlib._binding.onerror (zlib.js:295:17)
```

When I change the `tarball` path from `https://api.github.com/repos/my/private-packages/tarball/commithash` to `https://github.com//my/private-packages/tarball/commithash`, everything works as expected. Is there something I need to do to generate a tarball at that path in my repo? If so, that information should be included in the docs. If not, we should probably switch to suggesting the second type of tarball URL.

**Comment by creeonix** [comment]
would be nice to have something like:
```json
{
  "my:private-package": {
    "github": "my-repo/my-packages",
    "version": "lastest_or_0.15"
  }
}

**Comment by raix** [comment]
liike: https://github.com/raix/Meteor-famono#adding-additional-libraries :)

**Comment by raix** [comment]
missing from docs...:
```js
{
  "famous.polyfills": {
    "git": "https://github.com/Famous/polyfills.git",
    "tag": "v1.0.0"
  }
}
```

**Comment by jperl** [comment]
We are moving away from tarballs to using pure git. So this will be fixes in #19
